migrations/versions/15402497cb30_add_iam_download_permissions.py

```diff
@@ -8,7 +8,7 @@
 from alembic import op
 import sqlalchemy as sa
 
-from cidc_api.models import Permissions
+# from cidc_api.models import Permissions
```

Comment:
hm, and all commentings out below in this file

Reply:
I can rollback the prod + staging dbs and delete the migration, if that's preferable.

Reply:
Not sure what you mean, could you pls provide context? I'm a bit lost - rollback ? delete which\ migration?

Reply:
The file you've commented on here is a migration intended to automatically add IAM download permissions based on the records in the `permissions` table (this probably should've just been a one-off script one of us runs rather than an automatic migration, which is what we've decided to do to integrate updates from this PR). This migration has already been applied on both the staging and production dbs, so Alembic, the migration version manager we use, will encounter errors when next running migrations if this file is simply deleted. Rolling back (or `downgrade`-ing in Alembic terms) to the previous migration version in both environments will make it safe to delete this file, though.